tests/library/test_gen.py

```diff
@@ -52,10 +52,6 @@ def test_unicode(selected_model):
     lm + f'Step {i}:' + gen('steps', list_append=True, stop=['\nStep', '\n\n', '\nAnswer'], temperature=0.7, max_tokens=20) + '\n'
 
 def test_unicode2(selected_model):
-    # Does not work with Phi2
-    model_type = type(selected_model.engine.model_obj).__name__
```

Comment:
Since Phi2 is now fixed, incorporate the test re-enable into this PR (since it has to change the disabling mechanism anyway) @Harsha-Nori

Reply:
Seems reasonable